command/runner.go

```diff
@@ -203,3 +215,44 @@ func (r *cmdWaitReadCloser) Close() error {
 	r.closed = true
 	return nil
 }
+
+func runPager() (func(), error) {
+	var pager string
+	if p := os.Getenv("KUBEPAGER"); p != "" {
```

Comment:
Is there already a convention to use KUBEPAGER among Kubernetes tools? Otherwise you should use the `KUBECOLOR_` prefix

```suggestion
	if p := os.Getenv("KUBECOLOR_PAGER"); p != "" {
```

I also think this could also be added to the config package so it loads it from the config file. Such as:

```yaml
# ~/.kube/color.yaml

pager: less -R
```